docs/advanced_usage.rst

```diff
@@ -354,3 +354,183 @@ restrictions on the views. This can be overridden on a per-view basis as you wou
+
+.. class:: drf_haystack.serializers.HaystackSerializerMixin
+
+An example using the mixin might look like the following:
+
+.. code-block:: python
+
+    class PersonSerializer(serializers.ModelSerializer):
+        class Meta:
+            model = Person
+            fields = ("id", "firstname", "lastname")
+
+    class PersonSearchSerializer(HaystackSerializerMixin, PersonSerializer):
+        class Meta(PersonSerializer.Meta):
+            search_fields = ("text", )
+
+The results from a search would then contain the fields from the ``PersonSerializer`` rather than fields from the
+search index.
+
+.. note::
+
+    If your model serializer specifies a ``fields`` attribute in its Meta class, then the search serializer must
+    specify a ``search_fields`` attribute in its Meta class if you intend to search on any search index fields
+    that are not in the model serializer fields (e.g. 'text')
+
+.. warning::
+
+    It should be noted that doing this will retrieve the underlying object which means a database hit.  Thus, it will
+    not be as performant as only retrieving data from the search index.  If performance is a concern, it would be
+    better to recreate the desired data structure and store it in the search index.
+
+
+Multiple Search indexes
+=======================
+
+So far, we have only used one class in the ``index_classes`` attribute of our serializers.  However, you are able to specify
+a list of them.  This can be useful when your search engine has indexed multiple models and you want to provide aggregate
+results across two or more of them.  To use the default multiple index support, simply add multiple indexes the ``index_classes``
+list
+
+.. code-block:: python
+
+    class PersonIndex(indexes.SearchIndex, indexes.Indexable):
+        text = indexes.CharField(document=True, use_template=True)
+        firstname = indexes.CharField(model_attr="first_name")
+        lastname = indexes.CharField(model_attr="last_name")
+
+        def get_model(self):
+            return Person
+
+    class PlaceIndex(indexes.SearchIndex, indexes.Indexable):
+        text = indexes.CharField(document=True, use_template=True)
+        address = indexes.CharField(model_attr="address")
+
+        def get_model(self):
+            return Place
+
+    class ThingIndex(indexes.SearchIndex, indexes.Indexable):
+        text = indexes.CharField(document=True, use_template=True)
+        name = indexes.CharField(model_attr="name")
+
+        def get_model(self):
+            return Thing
+
+    class AggregateSerializer(HaystackSerializer):
+
+        class Meta:
+            index_classes = [PersonIndex, PlaceIndex, ThingIndex]
+            fields = ["firstname", "lastname", "address", "name"]
+
+
+    class AggregateSearchViewSet(HaystackViewSet):
+
+        serializer_class = AggregateSerializer
+
+.. note::
+
+    The ``AggregateSearchViewSet class above omits the optional ``index_models`` attribute.  This way results from all the
```

Comment:
Not to be picky, but a minor typo here. Seems you forgot to close the first quotes? Should probably be ``AggregateSearchView`` class above...